packages/frontend/src/hooks/echarts/useEcharts.ts

```diff
@@ -659,7 +660,8 @@ const useEcharts = () => {
     //Remove stacking from invalid series
     const stackedSeries = series.map((serie) => ({
         ...serie,
-        stack: serie.type === 'bar' ? serie.stack : undefined,
+        stack:
+            serie.type === 'bar' || !!serie.areaStyle ? serie.stack : undefined,
```

Comment:
@rephus I'd love to understand how we're using the echarts configuration here. What is `areaStyle`? And why is that sometimes set and other times not?

Reply:
Nice got it!